arcgis-ios-sdk-samples/Layers/Display annotation/README.metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "category": "Layers",
+    "description": "Display annotation from a feature service URL.",
+    "ignore": false,
+    "images": [
+        "DisplayAnnotation.png"
+    ],
+    "keywords": [
+        "Annotation",
```

Comment:
I'm not sure the capitalization of the keywords matter. For all of the other metadata files, we just copied the keywords exactly as they appeared in the README.